platforms/android/library/src/main/java/io/element/android/wysiwyg/EditorEditText.kt

```diff
@@ -616,6 +616,14 @@ class EditorEditText : AppCompatEditText {
         textWatcher.removeChild(watcher)
     }
 
+    override fun onSizeChanged(w: Int, h: Int, oldw: Int, oldh: Int) {
+        super.onSizeChanged(w, h, oldw, oldh)
+
+        // The size changed, so the cached positions for the code renderers won't match anymore
+        inlineCodeBgHelper.clearCachedPositions()
```

Comment:
Maybe this should be called before `super`?

Reply:
I tested this with the EX app and it worked fine, but it can't hurt to make it happen faster 👍 .